building/build-debs/homeworld-admin-tools/src/configuration.py

```diff
@@ -308,6 +310,13 @@ def get_machine_list_file() -> str:
     return ",".join("%s.%s" % (node.hostname, config.external_domain) for node in config.nodes) + "\n"
 
 
+def get_kube_cert_paths() -> (str, str, str):
+    project_dir = get_project()
+    return os.path.join(project_dir, "kube-access.key"),\
+           os.path.join(project_dir, "kube-access.pem"),\
+           os.path.join(project_dir, "kube-ca.pem")
+
+
 def get_local_kubeconfig() -> str:
     key_path, cert_path, ca_path = access.get_kube_cert_paths()
```

Comment:
Why does this point to `access.get_kube_cert_paths()` instead of `get_kube_cert_paths()`?